reconcile/utils/terraform_client.py

```diff
@@ -267,6 +267,7 @@ def log_plan_diff(
         always_enabled_deletions = {
             "random_id",
             "aws_lb_target_group_attachment",
+            "cloudflare_record",
```

Comment:
Maybe just worth a quick comment that zones can contain hundreds or more records, so it's not practical to require adding record to deletionApprovals?